apps/healthlocker/web/templates/page/privacy.html.eex

```diff
@@ -123,7 +123,7 @@
   <p>
     If you have any additional questions regarding this policy, please feel
     free to contact us at:
-  <%= link "Oxl-tr.focusadhd@nhs.net", to: "Oxl-tr.focusadhd@nhs.net", 
+    <%= link_to("Oxl-tr.focusadhd@nhs.net", to: "mailto:Oxl-tr.focusadhd@nhs.net") %>
```

Comment:
```link``` is the default Phoenix function to create html link
```link_to``` is a helper function created to attached some default classes to the link, see https://github.com/healthlocker/oxleas-adhd/blob/9a785f5e1b5a36879bf11f11385494a00c45812f/apps/healthlocker/web/views/component_helpers/link.ex